desktop/src/main/java/bisq/desktop/main/dao/monitor/StateMonitorView.java

```diff
@@ -102,11 +115,15 @@
     public StateMonitorView(DaoStateService daoStateService,
                             DaoFacade daoFacade,
                             CycleService cycleService,
-                            PeriodService periodService) {
+                            PeriodService periodService,
+                            SeedNodeRepository seedNodeRepository,
+                            File storageDir) {
```

Comment:
Should this be `@Named(Storage.STORAGE_DIR) File storageDir`?

Reply:
No its the base class, I will remove the Inject annotation and make it protected.